### PR TITLE
added a safety check concerning the downloaded kernel module for wifi…

### PR DESCRIPTION
**Pull request by rayjanwilson**
… support

### CODE REVIEW

**Comment by rayjanwilson** [comment]
closes #637, #567, #580

File: server.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ var logger = module.exports = new(winston.Logger)({
             level: process.env.GCS_LOG_LEVEL // if undefined, will be 'info'.
         }),
         new(winston.transports.File)({
-            filename: 'gcs.log',
+            filename: '/var/log/dapper-gcs/gcs.log',
```
Comment (brucecrevensten) [inline server.js]: adjust to use logging root in config file

File: hardware/README.md [hunk 2]
```diff
@@ -122,6 +122,10 @@ bcm_bt_lpm             13676  0
     - this also will take a bit
 - `grunt`
 
+### Create log directory
+- `mkdir -p /var/log/dapper-gcs`
+- `chmod 777 /var/log/dapper-gcs`
```
Comment (brucecrevensten) [inline hardware/README.md]: update to 755

File: etc/setup.sh [hunk 3]
```diff
@@ -57,15 +58,13 @@ Stage2 ()
...
-opkg install --force-reinstall ./kernel-module-bcm4334x_1.141-r47_edison.ipk
-echo "    ignore the FATAL, it's ok"
-
-
 #let's get back onto the wifi
 configure_edison --wifi
 
+echo "creating the log directory /var/log/dapper-gcs"
+mkdir -p /var/log/dapper-gcs
+chmod 777 /var/log/dapper-gcs
```
Comment (brucecrevensten) [inline etc/setup.sh]: update to 755

**Pull request merged**